.github/renovate.json

```diff
@@ -6,7 +6,7 @@
     ":dependencyDashboard",
     ":maintainLockFilesWeekly",
     ":pinAllExceptPeerDependencies",
-    ":prConcurrentLimit10",
+    ":prConcurrentLimit3",
```

Comment:
❌ There's only 10, 20, and none, so you'll need to copy https://github.com/bitwarden/clients/blob/master/.github/renovate.json#L13.